client_it_test.go

```diff
@@ -615,6 +615,8 @@ func TestClientStartShutdownMemoryLeak(t *testing.T) {
 			config.Logger.Level = logger.TraceLevel
 		}
 		config.Cluster.Unisocket = !smart
+		// TODO we should either decrease our default config values or find another way to process events more efficiently
+		// this will be temporary, fix this before PR merge
```

Comment:
@utku-caglayan Was this comment addressed?

Reply:
This is fixed, with out latest default values. However I forgot to remove this command. Thanks for pointing it out.